crux-llvm/README.md

```diff
@@ -34,16 +34,21 @@ The `crux-llvm` tool can be built by doing the following:
 * Change to the `crux-llvm` directory and run the build script:
 
         cd crucible/crux-llvm
-        ./build-stack.sh
+        cabal v2-build
 
-This will compile `crux-llvm` and supporting libraries and install them
-in the directory `crucible/crux-llvm/bin`.
+This will compile `crux-llvm` and supporting libraries such that they
+can be executed with `cabal v2-run`. To install the binaries in the
+standard Cabal binary path, run the following:
+
+        cabal v2-install exe:crux-llvm --project-file=install.project --overwrite-policy=always
```

Comment:
This branch already fixes the trickiest of the three errors in #552 - if you fixed the other two, then `install.project` wouldn't be needed, right?

Reply:
Heh. That's true. Maybe I should just fix the others and remove it.

Reply:
You may want to recommend a version of GHC/Cabal to install (e.g., 8.8.4 and 3.2).  I wonder if things work with the `2.*` version?

Also, for installing, it might be useful to mention `--installdir` this is where `cabal` will put the sym-link to the executable.